sdk/servicebus/azure-servicebus/samples/sync_samples/receive_deadlettered_messages.py

```diff
@@ -12,17 +12,31 @@
 # pylint: disable=C0111
 
 import os
-from azure.servicebus import ServiceBusClient
+from azure.servicebus import ServiceBusClient, Message
 
 CONNECTION_STR = os.environ['SERVICE_BUS_CONNECTION_STR']
 QUEUE_NAME = os.environ["SERVICE_BUS_QUEUE_NAME"]
 
 servicebus_client = ServiceBusClient.from_connection_string(conn_str=CONNECTION_STR)
 
 with servicebus_client:
-    receiver = servicebus_client.get_queue_deadletter_receiver(queue_name=QUEUE_NAME)
+    sender = servicebus_client.get_queue_sender(queue_name=QUEUE_NAME)
+    messages = [Message("Message to be deadlettered") for _ in range(10)]
+    with sender:
+        sender.send_messages(messages)
+
+    print('dead lettering messages')
+    receiver = servicebus_client.get_queue_receiver(queue_name=QUEUE_NAME)
     with receiver:
         received_msgs = receiver.receive_messages(max_batch_size=10, max_wait_time=5)
```

Comment:
Just as an observation; I don't see prefetch above, so this likely wouldn't have been happy regardless?  (Asking because it bumps the priority on my trying to get engsys to ingest these as the smoketests they run for their automation, to get two birds with one stone.)

Reply:
haha, you're being cheated.

In my dynamic link credit PR, I've already updated some samples removing the prefetch parameter which leads to your confusion here.

So this one is good actually :P